tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -1,8 +1,19 @@
 const { test, expect } = require( '@playwright/test' );
+const { createPage, deletePage } = require( '../../../utilities/rest-api' );
 const WpAdminPage = require( '../../../pages/wp-admin-page' );
 const EditorPage = require( '../../../pages/editor-page' );
 
 test.describe( 'Nested Tabs tests @nested-tabs', () => {
+	let pageId;
+
+	test.beforeEach( async () => {
+		pageId = await createPage();
+	} );
+
+	test.afterEach( async () => {
+		pageId = await deletePage( pageId );
```

Comment:
```suggestion
		await deletePage( pageId );
```

Reply:
Fixed in https://github.com/elementor/elementor/pull/21063/commits/3ccf1d714be4a48cf2fc2db219fb0b33b034c384